.github/CODEOWNERS

```diff
@@ -3,6 +3,8 @@
 
 # For more details, see https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
+/build @dockertopia @crazy-max
```

Comment:
I was waiting for the /build doc set to appear first to change the codeowners file, but thanks for this @crazy-max .